core/trino-main/pom.xml

```diff
@@ -473,7 +473,7 @@
         <dependency>
             <groupId>org.assertj</groupId>
             <artifactId>assertj-guava</artifactId>
-            <version>3.4.0</version>
+            <version>${dep.assertj-core.version}</version>
```

Comment:
We could start using `assertj-bom`

Reply:
https://github.com/airlift/airbase/pull/383